src/index.tsx

```diff
@@ -1,85 +1,50 @@
 import p5 from "p5";
-import React from "react";
+import deepEqual from "deep-equal";
+import React, { createRef, FC, memo, useEffect, useState } from "react";
 
-type TAttributes = { [key: string]: string };
-
-export interface IP5WrapperProps {
-  sketch: (p: p5) => void;
-  attributes: TAttributes;
+export interface Sketch {
+  (instance: p5): void;
 }
 
-export interface IP5WrapperState {
-  sketch: (p: p5) => void;
-  attributes: TAttributes;
-  canvas: p5;
-  wrapper?: HTMLElement;
+export interface P5WrapperProps {
```

Comment:
After changing the component name to `ReactP5Wrapper` should it also be applied here? `ReactP5WrapperProps`?